app/Repositories/AudioRepository.php

```diff
@@ -18,10 +18,17 @@ public function __construct(Audio $audio)
     public function getFormFields(TwillModelContract $audio): array
     {
         $fields = parent::getFormFields($audio);
+        $fields['title'] = $audio->title ?? $audio->getApiModel()?->title;
```

Comment:
FYI, on the website this is how we manage the `title` field on augmentable models: https://github.com/art-institute-of-chicago/artic.edu/blob/develop/app/Models/Behaviors/HasApiModel.php#L69-L83

Adding this and the related methods to `HasApiModel` would allow you to just call `$audio->title` here.